index.html

```diff
@@ -516,76 +477,62 @@ <h2 id="3Definitions">Definitions</h2>
 background.</dd>
 
 <!-- Maintain a fragment named "3criticalChunk" to preserve incoming links to it -->
-<dfn id="3criticalChunk">
-<dt>critical chunk</dt></dfn>
+<dt id="3criticalChunk"><dfn>critical chunk</dfn></dt>
 
-<dd><span class="Definition"><a>chunk</a></span>
+<dd><a>chunk</a>
 that <!--must be understood and processed by the decoder-->
  shall be understood and processed by the decoder in order to
-produce a meaningful image from a <span
-class="Definition"><a href="#4Concepts.Format"></a></span>.</dd>
+produce a meaningful image from a <a>PNG datastream</a>.</dd>
 
 <!-- Maintain a fragment named "3datastream" to preserve incoming links to it -->
-<dfn id="3datastream">
-<dt>datastream</dt></dfn>
+<dt id="3datastream"><dfn>datastream</dfn></dt>
 
-<dd>sequence of <span class=
-"Definition"><a>bytes</a></span>. This term is used rather than
+<dd>sequence of <a>bytes</a>. This term is used rather than
 "file" to describe a byte sequence that may be only a portion of
 a file. It is also used to emphasize that the sequence of bytes
 might be generated and consumed "on the fly", never appearing in
 a stored file at all.</dd>
 
 <!-- Maintain a fragment named "3deflate" to preserve incoming links to it -->
-<dfn id="3deflate">
-<dt>deflate</dt></dfn>
+<dt id="3deflate"><dfn>deflate</dfn></dt>
 
 <dd>name of a particular compression algorithm. This algorithm is
 used, in compression mode 0, in conforming
-<span class="Definition"><a>PNG
-datastreams</a></span>. Deflate is a member of the <a href=
-"#3LZ77"><span class="Definition">LZ77</span></a> family of
+<<a>PNG datastreams</a>. Deflate is a member of the <a>LZ77</a> family of
```

Comment:
There is a double `<<` at the beginning of this line. I don't think that was intentional.